File: demoSelenium.cs
using System;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;

namespace DemoSelenium.demoProject
{
    internal static class demoSelenium
    {
        public static void Main()
        {
            IWebDriver driver = new ChromeDriver();
            driver.Url = "http://www.5elementslearning.dev/demosite";
            driver.FindElement(By.LinkText("My Account")).Click();
            driver.Quit();
        }
    }
}

File: demoSelenium.java
package demoProject;
import org.openqa.selenium.*;
import org.openqa.selenium.chrome.*;
public class demoSelenium {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		driver.get("http://www.5elementslearning.dev/demosite");
		driver.findElement(By.linkText("My Account")).click();
		driver.quit();
	}

}
